apps/web-crescent/src/screens/validator_details/index.tsx

```diff
@@ -40,8 +40,8 @@ const ValidatorDetails = () => {
               data={votingPower}
               status={status.status}
             />
-            <Blocks className={classes.blocks} />
-            <Staking className={classes.staking} />
+            <Blocks className={classes.blocks} address={state.overview.operatorAddress} />
+            <Staking className={classes.staking} address={state.overview.operatorAddress} />
```

Comment:
Doubt here, this change is moved from `web-kyve` to `web-crescent`, is it not necessary in `web-kyve` any more?

Reply:
Yes it's not needed in `web-kyve` anymore

Reply:
And what is the reason it is needed in `web-crescent` but not in `web-kyve`?

Reply:
it's because we added `liquid staking` section in validator details page in `web-crescent` hence we need it here